### ISSUE TITLE
In VichImageType the wrong resolve method is used

### ISSUE THREAD

**Issue by c33s** [dustin10/VichUploaderBundle#871]
### Bug Report
|    Q        |   A
|------------ | ------
| BC Break    | ?
| Version     | 1.8.3

- javiereguiluz/easyadmin-bundle v1.17.12
- php7.1x64
- win7x64

#### Summary
TL;DR in `VichImageType` in `buildView` the `$this->storage->resolvePath` function should be changed to `$this->storage->resolveUri` for the liip imagine section. currently leading to a broken url on windows because the `FileSystemStorage` is using the in my eyes very unfamous `DIRECTORY_SEPERATOR`. 
```
http://example.com/media/cache/resolve/my_thumb/d/0/1%5Cfilename.jpg
```
see also https://github.com/EasyCorp/EasyAdminBundle/issues/2240

```
    public function buildView(FormView $view, FormInterface $form, array $options): void
    {
        $object = $form->getParent()->getData();
        $view->vars['object'] = $object;
        $view->vars['image_uri'] = null;
        $view->vars['download_uri'] = null;

        if (null !== $object) {
            if ($options['imagine_pattern']) {
                if (null === $this->cacheManager) {
                    throw new \RuntimeException('LiipImagineBundle must be installed and configured for using "imagine_pattern" option.');
                }

                $path = $this->storage->resolvePath($object, $form->getName(), null, true);
                if (null !== $path) {
                    $view->vars['image_uri'] = $this->cacheManager->getBrowserPath($path, $options['imagine_pattern']);
                }
            } else {
                $view->vars['image_uri'] = $this->resolveUriOption($options['image_uri'], $object, $form);
            }

            $view->vars = array_replace(
                $view->vars,
                $this->resolveDownloadLabel($options['download_label'], $object, $form)
            );

            $view->vars['download_uri'] = $this->resolveUriOption($options['download_uri'], $object, $form);
        }
        // required for BC
        // TODO: remove for 2.0
        $view->vars['show_download_link'] = !empty($view->vars['download_uri']);
    }
```
is there a special reason why `resolvePath` is used for liip imagine? if i change it to `resolveUri` it works like charm.

**sidenote**:
as long year windows user and php developer i only can say **do not use `DIRECTORY_SEPERATOR`** it doesn't help but only break things. use `/`
windows and php on windows is capable of opening files with the path `foo/bar\example/test.txt` 

but if you use `DIRECTORY_SEPERATOR`  which resolves to `\` on windows, things break.

```
λ cat ./foo/bar\example/test.txt
mycontent
```
(yes `cat` exists on windows)
```
notepad ./foo/bar\example/test.txt
```
also opens the file

`foo.php`:
```
<?php
$content = file_get_contents('foo/bar\example/test.txt');
var_dump($content);
```
```
λ php foo.php
C:\test\foo.php:5:
string(9) "mycontent"
```

**Comment by garak** [comment]
This has been already discussed in #858

**Comment by c33s** [comment]
closed in favor of #858
